Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -3911,6 +3919,16 @@ namespace AssetProcessor
 
         AZ::Uuid sourceUUID = AssetUtilities::GetSourceUuid(sourceAsset);
 
+        if (sourceUUID.IsNull())
+        {
+            auto failureMessage = AZStd::string::format("CreateJobs failed for %s - Invalid UUID, metadata file is likely corrupt", sourceAsset.AbsolutePath().c_str());
```

Comment:
are there .metadata files yet? If so, maybe print out the metadata file as well as the source asset file?

Reply:
Updated to include the metadata file path